src/synopsys-action/tools-parameter.ts

```diff
@@ -286,14 +286,15 @@ export class SynopsysToolsParameter {
     const githubToken = inputs.GITHUB_TOKEN
     const githubRepo = process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_REPOSITORY]
     const githubRepoName = githubRepo !== undefined ? githubRepo.substring(githubRepo.indexOf('/') + 1, githubRepo.length).trim() : ''
-    const githubBranchName = parseToBoolean(inputs.POLARIS_PRCOMMENT_ENABLED) ? process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_HEAD_REF] : process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_REF_NAME]
+    const githubBranchName = (parseToBoolean(inputs.POLARIS_PRCOMMENT_ENABLED) ? process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_HEAD_REF] : process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_REF_NAME]) || ''
     const githubRef = process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_REF]
-    const githubHostUrl = inputs.GITHUB_HOST_URL ? inputs.GITHUB_HOST_URL : ''
+    const githubServerUrl = process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_SERVER_URL]
+    const githubHostUrl = githubServerUrl === constants.GITHUB_CLOUD_URL ? undefined : githubServerUrl
```

Comment:
githubHostUrl will be undefined if GITHUB_SERVER_URL returns https://github.com
we should not pass anything like undefined to bridge